macros/WebExtCompat.ejs

```diff
@@ -1,34 +0,0 @@
-<%
```

Comment:
Ah, saw your comment in the original PR, "WebExtCompat will cause localized docs to break but not in en-US. I think localizers should be pinged about this." 👍